RakutenShop/Scenes/RecipeDetailView/RecipeDetailViewViewModel.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  RecipeDetailViewViewModel.swift
+//  RakutenShop
+//
+//  Created by Garnik Ghazaryan on 2/11/20.
+//  Copyright © 2020 Garnik Ghazaryan. All rights reserved.
+//
+
+import ReactiveSwift
+
+protocol RecipeDetailViewCoordinatorDelegate: class {
+    func didTapSaveButton()
+    func didTapInstructionTableViewCell(instructions: AnalyzedInstructionsResponseObject)
+    func didTapSimilarCollectionViewCell()
+}
+
+class RecipeDetailViewViewModel: BaseViewModel {
+
+    private let context: Context
+    weak var coordinatorDelegate: RecipeDetailViewCoordinatorDelegate?
+    
+    var recipeObject: MutableProperty<RecipeObject>?
+    
+    var nutritionsObject = MutableProperty<NutritionsResponseObject>(NutritionsResponseObject(calories: "", carbs: "", fat: "", protein: "", bad: [], good: []))
```

Comment:
var nutritionsObject = MutableProperty<NutritionsResponseObject?>(nil)
I believe this is more correct approach, as in case of very big responseObject with 20 fields, it would be problem for you.